### PR TITLE
Fix MSVC compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: mscore/excerptsdialog.cpp [hunk 1]
```diff
@@ -746,7 +746,7 @@ void ExcerptsDialog::accept()
 
       // Update the score parts order following excerptList widget
       // The reference is the excerpt list. So we iterate following it and swap parts in the score accordingly
-      for (int i = 0; i < excerptList->count(); ++i) {
+      for (i = 0; i < excerptList->count(); ++i) {
```
Comment (Howard-C) [inline mscore/excerptsdialog.cpp:749]: Maybe renaming it to `i1`? Because this `i` isn't related to the `i` above.

Comment (anatoly-os) [reply mscore/excerptsdialog.cpp:749]: Please, don't use numbers in the variable names. Use `j` if needed.

Comment (Jojo-Schmitz) [reply mscore/excerptsdialog.cpp:749]: There's no number, just 'recycling' an existing variable. But I can change that to j, of course, tomorrow ;-)

Comment (Jojo-Schmitz) [reply mscore/excerptsdialog.cpp]: done

**Pull request merged**